dc-commons-xml/src/main/java/de/digitalcollections/commons/xml/xpath/XPathMapper.java

```diff
@@ -229,4 +266,12 @@ private String resolveVariableContext(Locale language, String variableContext, M
     }
     return result;
   }
+
+  /**
+   * @param set A set with string elements
+   * @return true, when the set is empty or contains just a single empty string
+   */
+  private boolean isEmptyOrBlankStringSet(Set<String> set) {
+    return set.isEmpty() || ((set.size() == 1) && set.iterator().next().length() < 1);
```

Comment:
Nit: Lesbarer/Selbst-Dokumentierender mit Streams:

```suggestion
    return set.stream()
        .filter(val -> !String.isEmpty(val))
        .count() == 0;
```

Reply:
refactored now for better readability